dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/DigitalObjectRepositoryImpl.java

```diff
@@ -218,6 +329,58 @@ public void deleteFileResources(UUID digitalObjectUuid) {
     return fileResources;
   }
 
+  @Override
+  public List<LinkedDataFileResource> getLinkedDataFileResources(UUID digitalObjectUuid) {
+    final String ldfrTableAlias = linkedDataFileResourceRepositoryImpl.getTableAlias();
+    final String ldfrTableName = linkedDataFileResourceRepositoryImpl.getTableName();
+    final String fieldsSql = linkedDataFileResourceRepositoryImpl.getSqlSelectAllFields();
+    StringBuilder innerQuery =
+        new StringBuilder(
+            "SELECT dl.sortindex as idx, * FROM "
+                + ldfrTableName
+                + " AS "
+                + ldfrTableAlias
+                + " INNER JOIN digitalobject_linkeddataresources AS dl ON "
+                + ldfrTableAlias
+                + ".uuid = dl.linkeddata_fileresource_uuid"
+                + " WHERE dl.digitalobject_uuid = :uuid"
+                + " ORDER by idx ASC");
+    Map<String, Object> argumentMappings = new HashMap<>();
+    argumentMappings.put("uuid", digitalObjectUuid);
+
+    List<LinkedDataFileResource> linkedDataFileResources =
+        linkedDataFileResourceRepositoryImpl.retrieveList(
+            fieldsSql, innerQuery, argumentMappings, "ORDER BY idx ASC");
+
+    return linkedDataFileResources;
+  }
+
+  @Override
+  public List<FileResource> getRenderingFileResources(UUID digitalObjectUuid) {
```

Comment:
Das hat auch mit dem `DigitalObject` per se nicht viel zu tun und sollte eher ins `FileResourceMetadataRepo`, denke ich.

Reply:
Wird wegen der enormen Komplexität in einem nachfolgenden PR umgesetzt: https://github.com/dbmdz/cudami/pull/1585